GnuTLS returns the cipher strength in bytes and OpenLDAP expects bits.

Debian Bug#473796
Upstream ITS #5585

--- openldap.orig/libraries/libldap/tls.c
+++ openldap/libraries/libldap/tls.c
@@ -2796,7 +2796,7 @@
 	gnutls_cipher_algorithm_t c;
 
 	c = gnutls_cipher_get( session->session );
-	return gnutls_cipher_get_key_size( c );
+	return gnutls_cipher_get_key_size( c ) * 8;
 #else
 	return 0;
 #endif
